src/core/navbar/navbar-user/navbar-user.component.html

```diff
@@ -0,0 +1,73 @@
+<div *ngIf="!loggedIn()">
+  <div class="buttons has-addons">
+    <a class="button" (click)="authService.login(rememberLogin())">
+      <span class="icon-text">
+        <span class="icon">
+          <ng-icon name="remixLoginBoxLine"></ng-icon>
+        </span>
+        <span translate="core.login"></span>
+      </span>
+    </a>
+    <button
+      class="button has-tooltip-bottom"
+      data-tooltip="Remember Login"
+      id="remember"
+    >
+      <span
+        class="icon"
+        [class.has-text-primary]="rememberLogin()"
+        [class.has-opacity-60]="!rememberLogin()"
+        (click)="toggleRemember()"
+      >
+        <ng-icon size="1.6em" name="remixRotateLockFill"></ng-icon>
+      </span>
+    </button>
```

Comment:
The tooltip text is not fully visible on desktop clients with a width ≤ 1920px

![Bildschirmfoto am 2024-12-13 um 09 18 40](https://github.com/user-attachments/assets/90d76d1d-c9e0-4a75-bd7f-a17740643c47)

Reply:
Furthermore, the tooltip text is not available in German

Reply:
The tooltip is translated now, see 2472f160249f4e799778ecc57681df68a025fddd
The tooltip is now also right aligned with some padding to look more floaty, see a9888091a2b38f7474be290df4071b6f8d1a88bd